site/src/components/hero.md

```diff
@@ -114,6 +114,31 @@ On pages with right to left direction, the position of the hero image will remai
 </section>
 ```
 
+Hero images with details
+
+This component variant overlays a container the side opposite the hero content. It can be used to accommodate more content within the hero section if necessary.
```

Comment:
```suggestion
This component variant overlays a container on the opposite side of the hero content. It can be used to accommodate more content within the hero section if necessary.
```